pkgs/tools/networking/stubby/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, getdns, libtool, m4, file , doxygen
+, autoreconfHook, automake, check, libbsd, libyaml }:
+
+stdenv.mkDerivation rec {
+  pname = "stubby";
+  name = "${pname}-${version}";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "getdnsapi";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1cmv17wvc63rg4hx26b5a48l8jvlwc6w4003kyqbz4f961arz257";
+  };
+
+  nativeBuildInputs = [ libtool m4 libbsd libyaml autoreconfHook ];
+
+  buildInputs = [ doxygen getdns automake file check ];
+
+  meta = with stdenv.lib; {
+    description = "A local DNS Privacy stub resolver (using DNS-over-TLS)";
+    longDescription = ''
+Stubby is an application that acts as a local DNS Privacy stub
+resolver (using RFC 7858, aka DNS-over-TLS). Stubby encrypts DNS
+queries sent from a client machine (desktop or laptop) to a DNS
+Privacy resolver increasing end user privacy. Stubby is developed by
+the getdns team.
+'';
```

Comment:
Indentation, please, just as it was for getdns in the end https://github.com/NixOS/nixpkgs/pull/34921/files#diff-3f19f5b14f3dc0b7cdf83623bc28eb3cR25